USAGE.md

```diff
@@ -0,0 +1,142 @@
+```
+
+Install `generator-fountain-webapp`
+
+```sh
+$ npm install -g generator-fountain-webapp
+```
+Make a new directory, and cd into it:
+
+```sh
+mkdir my-new-project && cd $_
+``` 
+
+Run yo fountain-webapp:
+
+```sh
+yo fountain-webapp
+```
+
+## Yo options
+`yo fountain-generator --help` or `yo fountain-generator -h` for help. All options are not required. If not provided, default values will be used.
+
+* `--skip-cache` Do not remember prompt answers, default is `false`
+* `--skip-install` Do not automatically install dependencies, default is `false`
+* `--framework` Description for framework
+* `--modules` Description for modules manager
+* `--css` Description for css preprocessor
+* `--js` Description for js preprocessor
+
+## Use Gulp tasks
+
+- gulp or gulp build to build an optimized version of your application in /dist
+- gulp serve to launch a browser sync server on your source files
+- gulp serve:dist to launch a server on your optimized application
+- gulp test to launch your unit tests with Karma
+- gulp test:auto to launch your unit tests with Karma in watch mode
+
+## Directory structure
+
+Every component of the app get its own directory which contains the component code (depending on the framework, it can be `*.(js|ts)` or/and `*.component.(js|ts)` or/and `*.controller.(js|ts)` or/and `*.service.(js|ts)` or/and `*.directive.(js|ts)` and `*.spec.(js|ts)`.
+
+- If you are coding in Angular 1.4 style, you can read 
+[Best Practice Recommendations for Angular App Structure](https://docs.google.com/document/d/1XXMvReO8-Awi1EZXAXS4PzDzdNvV6pGcuaF4Q9821Es/pub). 
+- Concerning Angular ≥ 1.5 and Angular 2, you can read [Angular 2 Style Guide](https://mgechev.github.io/angular2-style-guide/). 
+- For React, you can read [Airbnb React Style Guide](https://github.com/airbnb/javascript/tree/master/react).
+
+<pre>
+├──  conf/
+│    ├──  browsersync-dist.conf.js
+│    ├──  browsersync.conf.js
+│    ├──  gulp.conf.js
+│    ├──  karma-auto.conf.js
+│    ├──  karma.conf.js
+│    ├──  ts.conf.json
+│    ├──  tslint.conf.json
+│    ├──  webpack.conf.js
+│    ├──  webpack-dist.conf.js
+│    ├──  webpack-test.conf.js
+│    └──  karma.conf.js
+│
+├──  gulp_tasks/
+│    ├──  browsersync.js
+│    ├──  build.js
+│    ├──  inject.js (optional)
+│    ├──  karma.js
+│    ├──  misc.js
+│    ├──  scripts.js
+│    ├──  styles.js
+│    ├──  systemjs.js (optional)
+│    └──  webpack.js (optional)
+│
+├──  bower_components/ (optional)
+├──  jspm_packages/ (optional)
+├──  nodes_modules/
+├──  typings/ (optional)
+│
+├──  src/
+│   ├──  app/
+│   │   ├── techs/
+│   │   │   ├──  tech.(js|ts)
+│   │   │   ├──  tech.spec.(js|ts)
+│   │   │   ├──  techs.(js|ts)
+│   │   │   └──  techs.spec.(js|ts)
+│   │   │
+│   │   ├── footer.(js|ts)
+│   │   ├── footer.spec.(js|ts)
+│   │   ├── header.(js|ts)
+│   │   ├── header.spec.(js|ts)
+│   │   ├── main.(js|ts)
+│   │   ├── main.spec.(js|ts)
+│   │   ├── title.(js|ts)
+│   │   └── title.spec.(js|ts)
```

Comment:
It implies the use of the Tech template. We can't put the directory structure of both Tech and Hello but perhaps put a warning on top.